docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -2,354 +2,336 @@
 title: Tic-Tac-Toe
 ---
 
-This guide covers three different implementations of the game tic-tac-toe on Sui. The first example utilizes a centralized admin that marks the board on the users’ behalf. The second example utilizes a shared object that both users can mutate. And the third example utilizes a multisig, where instead of sharing the game board, it's in a 1-of-2 multisig of both users’ accounts. This guide compares and contrasts the design philosophies behind the three different games, as well as the pros and cons of each.
+This guide covers three different implementations of the game tic-tac-toe on Sui. The first example utilizes a centralized admin that marks the board on the users’ behalf. The second example utilizes a shared object that both users can mutate. And the third example utilizes a multisig, where instead of sharing the game board, it's in a 1-of-2 multisig of both users’ accounts.
 
-## tic_tac_toe.move
+## owned.move
 
 In this first example of tic-tac-toe, the game object, including the game board, is controlled by a game admin.
 
 ```move
-public struct TicTacToe has key {
+public struct Game has key, store {
     id: UID,
-    gameboard: vector<vector<Option<Mark>>>,
-    cur_turn: u8,
-    game_status: u8,
-    x_address: address,
-    o_address: address,
+    board: vector<u8>,
+    turn: u8,
+    x: address,
+    o: address,
+    admin: vector<u8>,
 }
 ```
 
-Because the players don’t own the game board, they cannot directly mutate it. Instead, they indicate their move by creating a marker object with their intended placement and send it to the admin.
+Because the players don’t own the game board, they cannot directly mutate it. Instead, they indicate their move by creating a `Mark` object with their intended placement and send it to the game object using transfer to object:
 
 ```move
 public struct Mark has key, store {
     id: UID,
     player: address,
-    row: u64,
-    col: u64,
+    row: u8,
+    col: u8,
 }
 ```
-The main logic of the game is in the following `create_game` function.
+
+Games are created with the `new` function:
 
```

Comment:
{@inject: examples/tic-tac-toe/move/sources/owned.move#fun=new noComments}

I can do these on this PR or a separate one.

Reply:
Thanks @ronny-mysten ! I'd appreciate if you could show me how it's done this time around (in this PR is good). I wasn't sure what the status of the inject plugin was (whether it was ironed out, or whether there might be issues/edge cases), so I opted to focus on the content, but I'm happy for this to use it.

Couple of things that I thought might be edge cases:

- In one place, I reference only part of a function, with parts snipped out.
- I'm referencing code with a `.tsx` extension, which may complicate the logic of finding snippets.

Reply:
@amnn You're right. I added some features for move that aren't available for other langs yet (like removing comments). I also didn't consider cutting out more than one section at a time, but that should be a painless add. Anyway, I just updated the move parts for now.   

I left the copied code in there for now just so you can check it. That would obvs be deleted before publish. I also added the code collapsible for each `Source` link. I think both are valuable?

Reply:
Thanks very much @ronny-mysten ! And yes, I agree that both are valuable.

Reply:
One thing we need to tweak is tab sizing -- JS/TS code snippets tend to use tabs, and for me they are showing up as tab width 4, but I think folks tend to use tab-width 2 when editing them. I think it's as simple as setting a style to control tab size -- I'll look into it.